ls1intum/Artemis
# Adjust C tests used in K6 to new C template.

### Checklist

### Motivation and Context
The C template was updated in #2150, but the K6 tests were not updated. This resulted in failing K6 tests as the test code (from the template) and the submitted code (from the K6 script) mismatched.

### Description
Adjusted C template.

### Steps for Testing
1. Run K6 Tests on Bamboo (not Jenkins!) and verify that they pass: https://bamboo.ase.in.tum.de/browse/ARTEMIS-SAT3
